src/envo/envo-edit.owl

```diff
@@ -40086,6 +40089,53 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105010> "plastic pellet"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105010> <http://purl.obolibrary.org/obo/ENVO_01000776>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105011> (water column)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.merriam-webster.com/dictionary/water%20column>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105011> "A liquid astronomical body part which expands vertically between the surface and the floor of a body of water."@en)
```

Comment:
>"A liquid astronomical body part which expands vertically between the surface and the floor of a body of water.

Some issues here. 

The verb "expands" is ambiguous and it's unclear if a body of water has a floor (in a Newtonian sense, the ocean floor isn't part of the water body, but the basin that contains it)

I'd go for: "A liquid astronomical body part which 1) vertically spans a water body and 2) has fiat boundaries approximating a columnar shape."

We'd add PATO classes like [columnar](http://purl.obolibrary.org/obo/PATO_0002063) in the axioms during Phase II.